common/djangoapps/util/milestones_helpers.py

```diff
@@ -0,0 +1,136 @@
+"""
+Helper methods for milestones api calls.
+"""
+
+from collections import defaultdict
+from opaque_keys.edx.keys import CourseKey
+from milestones.api import (
+    get_course_milestones,
+    get_courses_milestones,
+    get_user_milestones,
+    add_milestone,
+    add_course_milestone,
+    remove_course_milestone,
+    get_course_milestones_fulfillment_paths,
+)
+from django.conf import settings
+
+
+def get_prerequisite_course_key(course_key):
+    """
+    Retrieves pre_requisite_course_key for a course from milestones app
+    """
+    pre_requisite_course_key = None
+    course_milestones = get_course_milestones(course_key=course_key, relationship="requires")
+    if course_milestones:
+        #TODO check if it is valid course key set as namespace
+        pre_requisite_course_key = course_milestones[0]['namespace']
+    return pre_requisite_course_key
+
+
+def add_prerequisite_course(course_key, prerequisite_course_key):
+    """
+    adds pre-requisite course milestone to a course
+    """
+    # create a milestone
+    milestone = add_milestone({
+        'name': 'Course {} requires {}'.format(unicode(course_key), unicode(prerequisite_course_key)),
+        'namespace': unicode(prerequisite_course_key),
+        'description': '',
```

Comment:
Since this is a system defined milestone, should we have a boiler-plate description here (and localized)?